atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -55,23 +52,26 @@ def __init__(self, tasker, workflow, parent_registry=None, parent_registry_insec
 
         self.check_platforms = check_platforms
         self.inspect_only = inspect_only
-        source_registry = get_source_registry(self.workflow, {
-            'uri': RegistryURI(parent_registry) if parent_registry else None,
-            'insecure': parent_registry_insecure})
-
-        if source_registry.get('uri'):
-            self.parent_registry = source_registry['uri'].docker_uri
-            self.parent_registry_insecure = source_registry['insecure']
-            self.parent_registry_dockercfg_path = source_registry.get('dockercfg_path', None)
-        else:
-            self.parent_registry = None
-            self.parent_registry_insecure = False
-            self.parent_registry_dockercfg_path = None
+
+        self.pull_registries = get_pull_registries(workflow, [])
```

Comment:
It doesn't look like this should be in `self` since it's only used within this method.

Reply:
right I'll delete 'self'